client/client.go

```
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"net"
+	"os"
+	"strings"
+)
+
+func main() {
+	clt, _ := net.Dial("tcp", "127.0.0.1:8083")
```

Please do not ignore an error returned by `Dial()` function.